h2o-extensions/xgboost/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     compile project(":h2o-genmodel")
     compile project(":h2o-genmodel-ext-xgboost")
     compileOnly 'com.esotericsoftware.kryo:kryo:2.21'
+    compile "org.apache.httpcomponents:httpclient:4.5.2"
```

Comment:
H2O-PERSIST-S3 and H2O-CORE seem to use older version. Can we also upgrade that one ? We'd then ship only one HTTPClient.

Reply:
good catch, fixed